meilisearch/src/analytics/segment_analytics.rs

```diff
@@ -786,6 +800,9 @@ impl SearchAggregator {
                    "avg_criteria_number": format!("{:.2}", self.filter_sum_of_criteria_terms as f64 / self.filter_total_number_of_criteria as f64),
                    "most_used_syntax": self.used_syntax.iter().max_by_key(|(_, v)| *v).map(|(k, _)| json!(k)).unwrap_or_else(|| json!(null)),
                 },
+                "attributes_to_search_on": {
+                   "total_number_of_use": self.attributes_to_search_on_total_number_of_use,
```

Comment:
```suggestion
                   "total_number_of_uses": self.attributes_to_search_on_total_number_of_use,
```

Reply:
If we're counting the number of times a user used the attribute, then we should probably rename to `total_number_of_uses` everywhere